source/system-administrators/activities/clustering/studio-clustering-two-nodes-with-arbiter.rst

```diff
@@ -1,6 +1,6 @@
 :is-up-to-date: True
 
-.. index:: Setup a Two Node Cluster with Studio and a Studio Arbiter, Auto-Clustering with Studio Example, Studio's Embedded Database Multi-master Cluster Example
+.. index:: Setup a Two Node Cluster with Studio and a Studio Arbiter, Clustering with Studio Example, Studio's Embedded Database Cluster Example
```

Comment:
Remove the words Embedded Database, it's just a Studio Cluster. Please do that throughout or ping me if you see a problem.